### DISCUSSION TITLE
LUA Script for opening a .txt file.

### DISCUSSION THREAD

**Discussion by inventor7777** [EdgeTX/edgetx#4973] [Q&A]
I wanted to make a Lua script that runs from the Tools menu, which simply opens up a text file (text_file.txt) inside the folder (FOLDER_ONE) which is also under the Tools menu. I've been trying to code it but I can't seem to figure out how to make it run without errors.

Can someone who knows more about Lua scripts help me? The radio is the 128x64 RadioMaster MT12.

**Comment by HinkePank** [comment]
In the current version ( 2,12,2 ) you can view .txt files from the system-menue by just simply selecting and choose open on them.
